spec.bs

```diff
@@ -2952,6 +2960,189 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a {{WindowOrWorkerGlobalScope/fetch()}} call using the
+{{RequestInit/adAuctionHeaders}} option, which causes the value of the [:Ad-Auction-Signals:]
+response header for that [=request=] to be used to provide signals in auctions running on the page

```

Comment:
> I think [=traversable navigable=] roughly coresponds to the notion of a page?

Yep. As mentioned in the comment thread above (which slightly overlaps with this one), traversable navigable is the right way of looking at this, and we should refer to individual documents inside of these, etc.

> Where would I specify the CORS behavior in the spec? I was sort of thinking this section was an introductory non-normative section, but I'm not sure how this is usually done.

You wouldn't specify it anywhere. Developers using `fetch()` would specify it. I'm just trying to make sure that our implementation isn't doing anything special with CORS on this kind of request that mismatches the spec (like overriding the CORS behavior or anything, to make the response always-readable).

Reply:
To follow-up here from some internal conversation, it seems that maybe:

> I know we discussed that currently no-cors requests still make the signals available to worklet functions (but not to the page). Is that an implementation bug that should be fixed?

...this might not be correct, since as I suspected, we are actually observing the headers of no-cors responses being empty (as I originally expected). So let's investigate to see why the browser test shows something different (i.e., shows no-cors request being delivered to the browser process interceptor with _non-empty_ headers).

Reply:
To follow up, as discussed offline, I filed crbug.com/1491934 to continue investigating the no-cors behavior in browser tests -- I've reached out to some folks who might know what's going on here.